src/Data/Formatter/Parser/Interval.purs

```diff
@@ -0,0 +1,87 @@
+module Data.Formatter.Parser.Interval
+  ( parseRecurringInterval
+  , parseInterval
+  , parseIsoDuration
+  , parseDateTime
+  , extendedDateTimeFormatInUTC
+  ) where
+
+import Prelude
+import Data.Interval as I
+import Text.Parsing.Parser as P
+import Text.Parsing.Parser.Combinators as PC
+import Text.Parsing.Parser.String as PS
+import Control.Alt ((<|>))
+import Data.Foldable (class Foldable, fold)
+import Data.Maybe (Maybe(..), maybe)
+import Data.Monoid (class Monoid, mempty)
+import Data.Either (Either, fromRight)
+import Data.Formatter.DateTime (unformatParser, Formatter, parseFormatString)
+import Data.DateTime (DateTime)
+import Data.Traversable (sequence)
+import Data.Tuple (Tuple(..), snd)
+import Partial.Unsafe (unsafePartialBecause)
+
+import Data.Formatter.Parser.Number (parseNumber, parseMaybeInteger)
+
+parseRecurringInterval :: ∀ a b. P.Parser String a -> P.Parser String b -> P.Parser String (I.RecurringInterval a b)
+parseRecurringInterval duration date =
+  I.RecurringInterval <$> (PS.string "R" *> parseMaybeInteger) <*> (PS.string "/" *> parseInterval duration date)
+
+parseInterval :: ∀ a b. P.Parser String a -> P.Parser String b -> P.Parser String (I.Interval a b)
+parseInterval duration date = [startEnd, durationEnd, startDuration, justDuration] <#> PC.try # PC.choice
+  where
```

Comment:
Could you please use same formatting rules for all files? E.g. we don't add indentation in `where` clause.